Packs/rasterize/Integrations/rasterize/rasterize.yml

```diff
@@ -3,320 +3,364 @@ commonfields:
   id: Rasterize
   version: -1
 configuration:
-- defaultvalue: 'false'
-  display: Return Errors
-  name: with_error
-  type: 8
-  required: false
-- defaultvalue: '0'
-  display: Time to wait before taking a screen shot (in seconds)
-  name: wait_time
-  type: 0
-  required: false
-- defaultvalue: '180'
-  display: Maximum time to wait for a page to load (in seconds)
-  name: max_page_load_time
-  type: 0
-  required: false
-- display: Chrome options (Advanced. See [?])
-  name: chrome_options
-  type: 0
-  additionalinfo: Add or remove Chrome options used to rasterize. Use for advanced troubleshooting. See Help.
-  required: false
-- display: Rasterize Mode
-  name: rasterize_mode
-  type: 15
-  defaultvalue: WebDriver - Preferred
-  options:
-  - WebDriver - Preferred
-  - WebDriver - Only
-  - Headless CLI - Preferred
-  - Headless CLI - Only
-  additionalinfo: It is possible to rasterize either via Chrome WebDriver or Chrome Headless CLI. A "Preferred" rasterizing mode will fall back to the other rasterizing mode if it fails.
-  required: false
-- display: Use system proxy settings
-  name: proxy
-  type: 8
-  required: false
+  - name: with_error
+    display: Return Errors
+    required: false
+    defaultvalue: "false"
+    type: 8
+    additionalinfo: null
+  - name: wait_time
+    display: Time to wait before taking a screen shot (in seconds)
+    required: false
+    defaultvalue: "0"
+    type: 0
+    additionalinfo: null
+  - name: max_page_load_time
+    display: Maximum time to wait for a page to load (in seconds)
+    required: false
+    defaultvalue: "180"
+    type: 0
+    additionalinfo: null
+  - name: chrome_options
+    display: Chrome options (Advanced. See [?])
+    required: false
+    defaultvalue: null
+    type: 0
+    additionalinfo: Add or remove Chrome options used to rasterize. Use for advanced
+      troubleshooting. See Help.
```

Comment:
In line 25 you wrote - See [?]
In line 30 you wrote - See Help.

Should they be the same?

Reply:
The `See [?]` (line 25) means that there's additional help text that appears in `additionalinfo` (line 30)

The `See Help` in line 30 refers to the file `rasterize_description.md` which I didn't change.